src/node/in_memory.rs

```diff
@@ -177,9 +179,10 @@ impl FromStr for ShowStorageLogs {
             "none" => Ok(ShowStorageLogs::None),
             "read" => Ok(ShowStorageLogs::Read),
             "write" => Ok(ShowStorageLogs::Write),
+            "paid" => Ok(ShowStorageLogs::Paid),
```

Comment:
What is `Paid`?

Reply:
storage logs slots for which you have to pay. Any ideas for a better names are welcome.

Reply:
This is AWESOME actually <3